### ISSUE TITLE
bug the current corpus.py ignores that status — it includes everything.

### ISSUE THREAD

**Issue by dreuxx** [tonghuikang/nemotron#1]
The previous corpus.py (the one you showed) had:
```

def choose_entry_to_include(problem_status, category):
    if category.endswith("_guess"):
        return True   # _guess are always included
    return problem_status == "rule_found"  # others: only if solved
```

**Comment by tonghuikang** [comment]
That was me iterating on the corpus.

I think it is good to include more training data when we are guessing anyway.

I want training loss to be a proxy for validation loss (training loss is validation loss if we are only training 1 epoch), including the data can allow me to reduce repeating the training data.
